doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -887,7 +887,9 @@ Common Application Framework
 Debug libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`cpu_load` library:
+
+  * Added prefix ``NRF_`` to all Kconfig options (for example, :kconfig:option:`CONFIG_NRF_CPU_LOAD`) to avoid conflict with Zephyr Kconfig option with the same name.
```

Comment:
```suggestion
  * Added prefix ``NRF_`` to all Kconfig options (for example, :kconfig:option:`CONFIG_NRF_CPU_LOAD`) to avoid conflicts with Zephyr Kconfig options with the same names.
```

Reply:
@divipillai

Reply:
Agree, as the fix is for multiple Kconfig options and their names.

Reply:
done